docs/production-deployment/cloud/terraform-provider.mdx

```diff
@@ -10,36 +10,34 @@ ssdi:
   - The Terraform Provider is in a Public Preview release status for Temporal Cloud.
 ---
 
-The Terraform Temporal provider is a plugin that allows Terraform to manage resources on Temporal Cloud.
-Terraform is a tool used to codify and provision infrastructure as code (IaC).
-With this provider, you can use Terraform to automate the management of Temporal Cloud Namespaces, resources for Temporal Cloud users, and more.
+The Terraform Temporal Cloud provider allows you to use Terraform to manage resources for Temporal Cloud.Terraform is a tool used to manage infrastructure as code (IaC). With this provider, you can use Terraform to automate the management of Temporal Cloud resources, like Namespaces, Users, Service Accounts, API Keys and more.
 
-The [Temporal Cloud Terraform provider](https://registry.terraform.io/providers/temporalio/temporalcloud/latest) is available in the Terraform Registry, where you can find detailed documentation on the supported resources and data sources.
+Remember, once a resource is managed by Terraform, you should only use Terraform to manage that resource.
```

Comment:
It might be worth bolding this, as this is often overlooked.

Reply:
TY - good call out